Tests/Functional/IndexQueue/Fixtures/fake_extension_tca.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+return [
```

Comment:
Please do not use short array syntax (yet).
If a change makes sense to be applied to the 3.1.x branch it needs to use explicit `array()` syntax as the tests for 3.1 are executed against PHP versions that do not support the short syntax.